src/components/pages/AdminHome/index.js

```diff
@@ -1,26 +1,17 @@
 import React from 'react';
 import '../../../styles/InstructorStyles/index.less';
 import AdminSidebar from './AdminSidebar';
-import AdminCalendar from './AdminCalendar';
+import AdminHomeContent from './AdminHomeContent';
 import { Layout } from 'antd';
-import Banner from '../../common/Banner';
 
 const { Content } = Layout;
 
```

Comment:
Let's remove `Content` if it is not being used.

Reply:
Done